packages/components/src/components/breadcrumb/breadcrumb.ts

```diff
@@ -0,0 +1,176 @@
+import { css, html } from 'lit';
+import { customElement } from '../../internal/register-custom-element';
+import { LocalizeController } from '../../utilities/localize';
+import { property, query, state } from 'lit/decorators.js';
+import cx from 'classix';
+import SolidElement from '../../internal/solid-element';
+
+/**
+ * @summary A responsive breadcrumb navigation components used to visualize a page's location
+ * within the site's hierarchy and provide easy navigation to previous sections.
+ * @status experimental
+ * @since 1.0
+ *
+ * @dependency sd-dropdown
+ * @dependency sd-icon
+ *
+ * @slot - `sd-breadcrumb-item` elements to display in the breadcrumb.
+ *
+ * @csspart base - The component's base wrapper.
+ * @csspart list - The list containing the slotted elements.
+ * @csspart truncated - The truncated wrapper.
+ * @csspart truncated-dropdown - The truncated dropdown containing the truncated breadcrumbs.
+ *
+ */
+@customElement('sd-breadcrumb')
+export default class SdBreadcrumb extends SolidElement {
+  public localize = new LocalizeController(this);
+
+  /** Internal resize observer */
+  private resizeObserver: ResizeObserver;
+
+  @query('[part="base"]') base: HTMLElement;
+  @query('[part="truncated"]') truncated: HTMLElement;
+  @query('[part="truncated-dropdown"]') dropdown: HTMLElement;
+  @query('slot') defaultSlot: HTMLSlotElement;
+
+  /** The breadcrumbs' label. Required for proper accessibility. */
+  @property({ type: String, reflect: true }) label = 'Breadcrumbs';
```

Comment:
I think we should either leave the label empty to force users to explicity set it OR if we want to set it ourselves we should localize it. What do you think?